### ISSUE TITLE
Replace React.createClass with createReactClass (react 15.5+)

### ISSUE THREAD

**Issue by fungilation** [chirag04/react-native-tooltip#30]
Chrome console output, on RN 0.46.2:
```
ExceptionsManager.js:71 Warning: React.createClass is no longer supported. Use a plain JavaScript class instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement. (https://fb.me/migrating-from-react-create-class)
reactConsoleErrorHandler | @ | ExceptionsManager.js:71
-- | -- | --
  | printWarning | @ | warning.js:36
  | warning | @ | warning.js:60
  | get | @ | react.development.js:2125
  | (anonymous) | @ | ToolTip.ios.js:24
  | loadModuleImplementation | @ | require.js:174
  | guardedLoadModule | @ | require.js:126
  | _require | @ | require.js:110
  | (anonymous) | @ | components.js:53
  | loadModuleImplementation | @ | require.js:174
  | guardedLoadModule | @ | require.js:126
  | _require | @ | require.js:110
  | (anonymous) | @ | index.ios.js:86
  | loadModuleImplementation | @ | require.js:174
  | guardedLoadModule | @ | require.js:119
  | _require | @ | require.js:110
  | (anonymous) | @ | redux-logger.js:1
  | executeApplicationScript | @ | debuggerWorker.js:40
  | (anonymous) | @ | debuggerWorker.js:65
```

A sample PR to fix: https://github.com/maxs15/react-native-modalbox/pull/148/files

**Comment by fungilation** [comment]
A lingering warning still on RN 0.47.1:
```
ExceptionsManager.js:73 Warning: PropTypes has been moved to a separate package. Accessing React.PropTypes is no longer supported and will be removed completely in React 16. Use the prop-types package on npm instead. (https://fb.me/migrating-from-react-proptypes)
reactConsoleErrorHandler @ ExceptionsManager.js:73
printWarning @ warning.js:35
warning @ warning.js:59
get @ react.development.js:2133
(anonymous) @ index.js:101
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ index.js:1
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ components.js:30
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ index.ios.js:88
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:119
_require @ require.js:110
(anonymous) @ redux-logger.js:1
executeApplicationScript @ debuggerWorker.js:40
(anonymous) @ debuggerWorker.js:65
```
```
ExceptionsManager.js:73 Warning: React.createClass is no longer supported. Use a plain JavaScript class instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement. (https://fb.me/migrating-from-react-create-class)
reactConsoleErrorHandler @ ExceptionsManager.js:73
printWarning @ warning.js:35
warning @ warning.js:59
get @ react.development.js:2125
(anonymous) @ ToolTip.ios.js:24
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ components.js:53
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:126
_require @ require.js:110
(anonymous) @ index.ios.js:88
loadModuleImplementation @ require.js:174
guardedLoadModule @ require.js:119
_require @ require.js:110
(anonymous) @ redux-logger.js:1
executeApplicationScript @ debuggerWorker.js:40
(anonymous) @ debuggerWorker.js:65
20:13:15.465 
```

Unless you are telling me you can't reproduce these warnings in any RN >= 0.45 project?
![screenshot 2017-08-03 20 21 15](https://user-images.githubusercontent.com/3803466/28952928-75fd2ef0-7889-11e7-9d7b-70c322fd804b.png)
In [project]/node_modules/react-native-tooltip/ToolTip.ios.js, the `ToolTip.ios.js` referenced in above log:
![screenshot 2017-08-05 12 10 25](https://user-images.githubusercontent.com/3803466/28998070-588fda62-79d7-11e7-896c-752b09c0f08c.png)

**Comment by enahum** [comment]
I believe this os no longer the case, can you please try with the latest version?

**Comment by fungilation** [comment]
Yup, all (my) issues seems fixed in latest 5.2.0. Reinstalled `react-native-tooltip` from latest in npm, thanks
